community-membership.md

```diff
@@ -19,6 +19,9 @@ scoped by these sub-areas.
 [**New contributors**](https://github.com/kubernetes/contrib/issues/1090) should be welcomed to the community
 by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
 
+**Note:** Anyone can be added as an outside collaborator (with READ access) to a repo in the Kubernetes GitHub organization
```

Comment:
There still needs to be a process for this, and some nominal requirements.

I believe only repo admins and org owners can add outside collaborators.